package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Mon Nov 20 15:40:58 UTC 2017 - shundhammer@suse.com
+
+- Added "Create New Partition Table" in partitioner
+
```

Comment:
There is no fate number. I asked for one yesterday, and I got the same response that I mostly get when asking anything: Zero. This is also why I did not bump the version number because then one of our check bots will complain.

Reply:
Ok, sorry, I didn't see your question yesterday. As commented, we are simply putting "part of fate#318196" (generic fate for storage reimplementation) when we don't have a specific fate/bug number.

Reply:
OK, that seems like a good workaround.